Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -271,6 +292,18 @@ private async UniTaskVoid SearchAndSetEmojiSuggestionsAsync(string value, Cancel
             emojiSuggestionPanelController.SetPanelVisibility(true);
         }
 
+        private void CloseAllEmojiPanels(InputAction.CallbackContext obj)
+        {
+            eventSystem.RaycastAll(Mouse.current.position.value);
```

Comment:
Why do you raycast without the result consumption? Probably a leftover?

Reply:
leftover yeah